amis/packer_ubuntu1804.json

```diff
@@ -210,6 +198,15 @@
         "sudo /opt/cinc/embedded/bin/gem install --no-document berkshelf:{{user `berkshelf_version`}} ffi-libarchive"
       ]
     },
+    {
+      "type" : "shell",
+      "inline" : [
+        "set -x",
+        "sudo mkdir -p /etc/chef/cookbooks",
+        "sudo /opt/cinc/embedded/bin/berks vendor /etc/chef/cookbooks --berksfile /tmp/cookbook/Berksfile",
```

Comment:
`/etc/chef/cookbooks` is created by this command if it doesn't already exists?

Reply:
yes, I'm removing the mkdir